pkg/apis/kubeone/types.go

```diff
@@ -524,6 +524,8 @@ type RegistryConfiguration struct {
 	// in OverwriteRegistry as an insecure registry. This is also propagated
 	// to the worker nodes managed by machine-controller and/or KubeOne.
 	InsecureRegistry bool `json:"insecureRegistry,omitempty"`
+	// Mirrors sets registry-mirrors
```

Comment:
Please expand this comment a little bit to include some more details on what are registry mirrors.